sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -1863,6 +1886,29 @@ public async Task GetPropertiesAsync_AccessConditionsFail()
             }
         }
 
+        [Test]
+        public async Task GetPropertiesAsync_ObjectReplication()
```

Comment:
Let's add TODOs on these tests to figure out what we want to do with handling setting ORS on containers.